lib/ret/entity_state/create_message.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Ret.EntityState.CreateMessage do
```

Comment:
This is a potential confusing struct name, being both a noun and a predicate. It’s conventional for Elixir functions that retrieve an indeterminate number of elements to be named with a collective noun (e.g., `potatoes/0`). If we were to do that here (i.e., `create_messages/0`), it wouldn’t be unreasonable for the reader to infer that the function is a command rather than a query.

Reply:
I don’t understand the name, but maybe `MessageCreation` is better?